Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -939,3 +940,16 @@
       components:
       - SurplusBundle
+
+- type: listing
+  id: UplinkSelfRechargeBattery
+  name: uplink-microreactor-name
+  description: uplink-microreactor-desc
+  productEntity: PowerCellMicroreactor
+  cost:
+    Telecrystal: 35
+  categories:
+  - UplinkDisruption
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)